omod/src/main/resources/org/openmrs/module/fhir2/narratives/AllergyIntolerance.html

```diff
@@ -81,13 +81,13 @@
             <td>Onset:</td>
             <td th:narrative="${resource.onset}"></td>
         </tr>
-        <th:block th:if="${fhirVersion} eq 'DSTU3'">
+        <th:block th:unless="${fhirVersion} eq 'R4'">
```

Comment:
`assertedDate` is only available in R3. R5 and beyond presumably use the R4 property names by default.

Reply:
Okay so I will change it back to `th:if="${fhirVersion} eq 'DSTU3'"`.